charmstore_test.go

```diff
@@ -506,33 +506,41 @@ func (s *charmStoreRepoSuite) TestResolve(c *gc.C) {
 
 	// Define the tests to be run.
 	tests := []struct {
```

Comment:
Can we have at least one test that tests the results for a multi-series charm, please?